examples/30_extended/datasets_tutorial.py

```diff
@@ -52,7 +52,7 @@
 # =================
 
 # This is done based on the dataset ID.
```

Comment:
```suggestion
```

Reply:
For this particular example, which actually is a tutorial for datasets, it probably makes sense to comment more on the fact that they can be downloaded with an id which can be an integer or a name and then explaining a bit more that a name would be good to be followed up with a version since the datasets can change between versions.

Reply:
That's fair but the comment did not reflect that. So in that case I think it should be modified to something like `This can also be done based on the dataset ID.` probably accompanied by a commented out example `# dataset = openml.datasets.get_dataset(1471)`

Reply:
Yes, you are right on that. This is also what I was thinking about.